src/main/logger.ts

```diff
@@ -48,4 +49,55 @@ const logger = winston.createLogger({
   ],
 });
 
-export default logger;
+type Logger = ReturnType<typeof winston.createLogger>;
+type LoggerKeys = keyof Logger;
+type LoggerError = Logger["error"];
+
+/**
+ * Type guard to ensure unknown is logger.error function
+ */
+const isLoggerError = (unknown: unknown, key: LoggerKeys): unknown is LoggerError =>
```

Comment:
nit using `unknown` as an arg name is confusing

Reply:
Fixed in https://github.com/lensapp/lens/pull/3364/commits/146891e4315bf5315745753f37101a769009bbbd